udata/tests/forms/test_reference_field.py

```diff
@@ -7,7 +7,7 @@
 from udata.forms import ModelForm, fields
 from udata.models import db
 from udata.tests import TestCase
-from udata.tests.factories import faker
+from udata.core.user.factories import faker
```

Comment:
In https://github.com/etalab/udata/pull/444/commits/1b3226924f7aa46608a6e56aae04d0de4532e91f#diff-718f5796155236b92d81115cf73d992cR208 faker was moved to `udata.utils`